modules/ROOT/pages/6.0-release-notes-premium-changes.adoc

```diff
@@ -53,6 +104,39 @@ No Premium Plugins released with {productname} 6.0 are backwards compatible with
 
 The following premium plugin updates were released alongside {productname} 6.0.
 
+=== Accessibility Checker
+
+The default value for `a11ychecker_html_version` was `html4`.
+
+It is now `html5`.
+
+=== Advanced Table
+
+The `getRowType` function would sometimes incorrectly parse elements in the `tfoot` section of a table.
+
+This led to `mceTableRowType` being incorrectly returned as a header if all cells in the section were `th` elements.
```

Comment:
@ltrouton Could you please check this bit? It reads a bit odd to me, and you're the expert.

Reply:
@brian-forte The information in the `Advanced Table` subsection isn't quite correct so would have to be reworked if we wanted to keep it. 

I am curious why this subsection was added as this is more of a basic fix and there were other fixes/changes made for the `advtable` plugin as well as part of 6.0. Apologies if this information is somewhere else that I haven't seen.

Reply:
> The information in the Advanced Table subsection isn’t quite correct
> so would have to be reworked if we wanted to keep it.

What’s the nature of the error in the current text?

> I am curious why this subsection was added as this is more of a
> basic fix and there were other fixes/changes made for the `advtable`
> plugin as well as part of 6.0.

It’s the only change to `advtable` that I had any info on (via [this portion](https://www.notion.so/tinycloud/6-0-Docs-Notes-94ee2ecf10ff440b9246a22f8cc54828#3e4cd38137354dcd8fbe92d7dc7c5eaf) of [this Notion](https://www.notion.so/tinycloud/6-0-Docs-Notes-94ee2ecf10ff440b9246a22f8cc54828) doc).

Reply:
Above questions answered via Slack DM.